src/transformers/models/wav2vec2_phoneme/tokenization_wav2vec2_phoneme.py

```diff
@@ -311,6 +397,7 @@ def _decode(
         group_tokens: bool = True,
         filter_word_delimiter_token: bool = True,
         spaces_between_special_tokens: bool = False,
+        output_char_offsets: Optional[bool] = False,
```

Comment:
The bool is not optional here, since it defaults to False, not None.
```suggestion
        output_char_offsets: bool = False,
```